domain/venue/venue_repository.py

```diff
@@ -12,3 +12,7 @@ def find_by_siret(self, siret: str) -> Venue:
     @abstractmethod
     def find_by_name(self, name: str, offerer_id: str) -> List[Venue]:
         pass
+
+    @abstractmethod
+    def get_all_by_pro_identifier(self, pro_identifier: int) -> List[Venue]:
```

Comment:
```suggestion
    def find_by_pro_identifier(self, pro_identifier: int) -> List[Venue]:
```
Si on veux appliquer un JPA style ^^

Reply:
On avait réfléchi à find vs get
Vu que dans ce cas là ne pas avoir de résultat est un cas normal (on reçoit une liste vide), on a préféré un `get`

Un `find` on s'attend à avoir nécessairement des résultats

Par contre en effet le `all` peut être enlevé